### PR TITLE
Errorboundary

### PR DESCRIPTION
**Pull request by EstebanIndiveri**
### Summary
Added a new CustomError,
Added a CustomModal
Added a CustomModalConfirm
Added a withpostFailure SignUp
Added a withPostFailure Login
Added a new message to withPostSucces SignUp
Changed Auth Actions replace extensión with TSX because need React 
Added feedback Redux and actions = ShowModal and HideModal to show custom Modals

### IOS 
Captura de Pantalla 2021-09-03 a la(s) 19 13 05
Captura de Pantalla 2021-09-03 a la(s) 19 14 50
Captura de Pantalla 2021-09-03 a la(s) 19 19 22
Simulator Screen Recording - iPhone 12 - 2021-09-03 at 19 25 05

###Android
Captura de Pantalla 2021-09-03 a la(s) 19 36 11
Captura de Pantalla 2021-09-03 a la(s) 19 37 11

### CODE REVIEW

File: src/app/components/CustomModal/styles.ts [hunk 1]
```diff
@@ -0,0 +1,34 @@
+import { darkGray, white } from '@constants/colors';
+import { StyleSheet } from 'react-native';
+
+const commonMargin = 16;
```
Comment (andreszuluaga-wolox) [inline src/app/components/CustomModal/styles.ts:4]: use upercase to constants

**Review by andreszuluaga-wolox — APPROVED** [review]

File: src/app/components/CustomModal/styles.ts [hunk 2]
```diff
@@ -0,0 +1,34 @@
+import { darkGray, white } from '@constants/colors';
+import { StyleSheet } from 'react-native';
```
Comment (john07289) [inline src/app/components/CustomModal/styles.ts:2]: move this to first line

File: src/app/components/CustomModalConfirm/index.tsx [hunk 3]
```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { View } from 'react-native';
+
+import CustomButton from '../CustomButton';
```
Comment (john07289) [inline src/app/components/CustomModalConfirm/index.tsx:4]: @components/CustomButton

File: src/app/components/CustomModalConfirm/index.tsx [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+import CustomText from '../CustomText';
```
Comment (john07289) [inline src/app/components/CustomModalConfirm/index.tsx:5]: @components/CustomText

File: src/app/components/CustomModalConfirm/styles.ts [hunk 5]
```diff
@@ -0,0 +1,24 @@
+import { extraLightGray, white } from '@constants/colors';
+import { SPACING } from '@constants/dimentions';
+import { StyleSheet } from 'react-native';
```
Comment (john07289) [inline src/app/components/CustomModalConfirm/styles.ts:3]: move this to first line

**Review by john07289 — APPROVED** [review]

**Pull request merged**